packages/ses/src/module-instance.js

```diff
@@ -136,6 +136,7 @@ export const makeModuleInstance = (
     __liveExportMap__: liveExportMap = {},
     __reexportMap__: reexportMap = {},
     __needsImportMeta__: needsImportMeta = false,
+    __precompiledFunctor__,
```

Comment:
The terms of `__syncModuleProgram__` mean:

- `sync` because it doesn’t support top-level-await, but someday `async` should
- `module` because it was a [[Module]]
- `program` because it is a [[Script]]
- `__` because it isn’t going to work without the shim

I’d like to consider naming this `__syncModuleFunctor__`
- `sync` because `async` will support top-level-await maybe someday
- `module` because reasons
- `functor` instead of `program` because the grammar production is irrelevant, the calling convention is.